ftl/core/browsing.ftl

```diff
@@ -70,14 +76,11 @@ browsing-question = Question
 browsing-queue-bottom = Queue bottom: { $val }
 browsing-queue-top = Queue top: { $val }
 browsing-randomize-order = Randomize order
-browsing-remove-current-filter = Remove Current Filter...
-browsing-remove-from-your-saved-searches = Remove { $val } from your saved searches?
 browsing-remove-tags = Remove Tags...
 browsing-replace-with = <b>Replace With</b>:
 browsing-reposition = Reposition...
 browsing-reposition-new-cards = Reposition New Cards
 browsing-reschedule = Reschedule
-browsing-save-current-filter = Save Current Filter...
```

Comment:
I think this and `browsing-remove-current-filter` are remnants from the filter button, so I removed them. Correct me if I'm wrong.

Reply:
Tests should fail if you accidentally delete a key that's used, so we should be good :-)